gherkin-languages.json

```diff
@@ -3134,15 +3134,15 @@
   "ta": {
     "and": [
       "* ",
-      "மேலும்  ",
+      "மேலும் ",
       "மற்றும் "
     ],
     "background": [
       "பின்னணி"
     ],
     "but": [
       "* ",
-      "ஆனால்  "
+      "ஆனால் "
```

Comment:
How did you spot these?

Reply:
Tried the Irish implementation, observed the issue, and then ran a check against all keywords where we typically expect a trailing space.

There were other occurrences (such as for emojis and CJK characters) though I wouldn't know the expectation in these cases; particularly with [valid use cases such as French](https://github.com/cucumber/gherkin/blob/f5cbb5d7f52de872eb881526ea5eff4a980bbd7e/gherkin-languages.json#L1361) where we'd avoid a trailing space.

We could potentially add to each language test job (or standalone job) to check a deep match against the source of truth language configuration. Unsure of the tradeoffs, but would be a nice peace of mind to automate alignment.

Good shout - I'll introduce an additional changelog entry - thanks!